g3proxy/docker/lua.alpine.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM nickblah/lua:5.4-luarocks-alpine AS lua
+RUN apk add --no-cache build-base
+# install lua modules here.
+# dkjson is just an example.
+# beware: lua's c modules (e.g. cjson)
+# need to be statically linked
+# and won't work when installing in
+# alpine via luarocks. 
+RUN luarocks install dkjson
+
+FROM rust:alpine AS builder
+WORKDIR /usr/src/g3
+COPY . .
+RUN apk add --no-cache musl-dev cmake capnproto-dev openssl-dev openssl-libs-static c-ares-dev c-ares-static lua5.4-dev pkgconfig
+ENV PKG_CONFIG_PATH=/usr/lib/pkgconfig
+RUN cargo build --profile release-lto \
+ --no-default-features --features quic,c-ares,hickory,lua54 \
+ -p g3proxy -p g3proxy-ctl
```

Comment:
You may also want to compile in g3proxy-lua, which can be used to verify the call result of the script

Reply:
Append '-p g3proxy-lua'?

Reply:
yes

Reply:
okay you got it, I'll add that tonight after testing it.

Reply:
I've added the bit of code but it doesn't work.

Inside the container, after rebuilding and running:
```bash
ls /usr/bin/g3proxy
g3proxy      g3proxy-ctl
```

There's no g3proxy-lua binary.

Reply:
COPY is also needed in the final build step.